### DISCUSSION TITLE
[Combobox]: Add support for multiple values

### DISCUSSION THREAD

**Discussion by knoefel** [tailwindlabs/headlessui#1164] [Ideas]
First, thank you for adding a `<Combobox />` component!!!

Right now it is only possible to select a single value. It would be nice to have the option to select multiple values.

Is this something that is already planned?

Thanks in advance and keep up the great work!!!

**Comment by knoefel** [comment]
Sorry, just saw there is already a discussion regarding this. I only searched for issues not discussions. 😳

**Comment by hussamkhatib** [comment]
https://headlessui.dev/react/combobox#selecting-multiple-values
